TeslaMateAgile/Services/BarryService.cs

```diff
@@ -79,7 +79,7 @@ public class BarryResponse
     public class BarryRequest
     {
         [JsonPropertyName("method")]
-        public string Method { get; set; } = "co.getbarry.api.v1.OpenApiController.getTotalKwHourlyPrice";
+        public string Method { get; set; } = "co.getbarry.api.v1.OpenApiController.getPrice";
```

Comment:
What's the purpose of this change?

Reply:
They changed their API after they closed down. One could still ask for prices for older dates. Probably not the right spot for this as it has nothing to do with this PR.